docs/source/conf.py

```diff
@@ -21,6 +21,14 @@
 
 import pt_lightning_sphinx_theme
 
+# -----------------------
+# VARIABLES WHEN WORKING ON DOCS... MAKE THIS TRUE TO BUILD FASTER
+# -----------------------
+_FAST_DOCS_DEV = False
```

Comment:
This is great! Two suggestions here:
1. We could expose an env variable so that users could do `PL_FAST_DOCS_DEV=1 make html` while building docs rather than updating the `conf.py` file themselves.
2. Adding a `PL` prefix to it, so that it indicates it's PyTorch Lightning related.

```suggestion
_PL_FAST_DOCS_DEV = bool(int(os.getenv("PL_FAST_DOCS_DEV", 0)))
```